domain/errors.go

```diff
@@ -0,0 +1,7 @@
+package domain
+
+import "github.com/pkg/errors"
+
+var (
+	ErrNoSuchEntity = errors.New("error no such entity")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var ErrNoSuchEntity should have comment or be unexported